onnxscript/onnx_opset/_impl/opset_ai_onnx_ml2.py

```diff
@@ -0,0 +1,82 @@
+# --------------------------------------------------------------------------
+# ⚠️ WARNING - AUTO-GENERATED CODE - DO NOT EDIT ⚠️
+# ⚙️ Generated by: setup.py opgen
+# --------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+# --------------------------------------------------------------------------
+
+from typing import Sequence, Union
+
+from onnx.defs import get_schema
+
+from ...onnx_types import FLOAT, INT64, STRING
+from ...values import Op, Opset
+from .opset_ai_onnx_ml1 import Opset_ai_onnx_ml1
+
+
+class Opset_ai_onnx_ml2(Opset_ai_onnx_ml1):
+    def __new__(cls):
+        return Opset.__new__(cls, "ai.onnx.ml", 2)
+
+    def __init__(self):
+        super().__init__()
+
+    def LabelEncoder(
+        self,
+        X: Union[FLOAT, INT64, STRING],
+        default_float: float = -0.0,
+        default_int64: int = -1,
+        default_string: str = "_Unused",
+        keys_floats: Sequence[float] = None,
+        keys_int64s: Sequence[int] = None,
+        keys_strings: Sequence[str] = None,
+        values_floats: Sequence[float] = None,
+        values_int64s: Sequence[int] = None,
+        values_strings: Sequence[str] = None,
+    ) -> Union[FLOAT, INT64, STRING]:
+        """
+        Maps each element in the input tensor to another value.<br>
```

Comment:
I wonder how this one renders since the indentation is weird and it includes html tags.

Reply:
They're now run through the same function as ONNX's docsgen.